docs/application/tizen-studio/release-notes/3-5-release-notes.md

```diff
@@ -0,0 +1,77 @@
+# Tizen Studio 3.5 Release Notes
+
+-  Release Date: Oct 30, 2019
+
+
+## IDE and Tools
+
+### New Features
+-   IDE
+	-   Oracle JDK 12 and OpenJDK 12 support have been added to make the IDE and tools work with Java 12.
+	-   Project creation wizard has been improved to add Flat View. Flat View provides a different view while creating a project.
+	-   Support for the Add-On type application has been added. Add-On type is a new template for Web applications from Tizen 5.5.
+	-   Support for the component-based application has been added. Component-based application is a new template for Tizen applications from Tizen 5.5.
+	-   Support for adding dependent package information provided in config and manifest editor from Tizen 5.5.
+	-   Support for "api-version" attribute in application manifest from Tizen 5.5.
+	-   Support for "visibility" attribute in app-control element from Tizen 5.5.
+
+### Fixed Bugs
+-   IDE
+	-   The issue of Samsung Certificate unable to sign application while launching an application on Tizen 5.0 Emulator has been fixed.
+
+### Deprecated
+-   IDE
+	-   Java 9 and OpenJDK 10 support has been deprecated.
+	-   Log Viewer has been deprecated.
+	-   32 bit systems support has been deprecated.
```

Comment:
Please rephrase to:
32-bit system support has been deprecated.

Please verify.